Tests/Marketplace/marketplace_services.py

```diff
@@ -1636,6 +1637,10 @@ def collect_content_items(self):
                             'name': content_item.get('name', ""),
                             'description': content_item.get('description', ""),
                         })
+                    elif current_directory == PackFolders.PREPROCESS_RULES.value:
+                        folder_collected_items.append({
+                            'name': content_item.get('scriptName', "") #TODO: verify, couldnt find in scheme
+                        })
```

Comment:
```suggestion
                            'name': content_item.get('scriptName', "")
                        })
```
what about description?

Reply:
PP rule was not added to the scheme yet, will wait with this PR until the scheme is clear.

Reply:
Verified by the scheme in the pp rule server PR